### ISSUE TITLE
2.2.0, changelog and PR management

### ISSUE THREAD

**Issue by jblaise** [angular-ui-tree/angular-ui-tree#354]
Hi,

First, thanks for your work on this module.

Like #351, my app  was (briefly) broken by the new way of passing callbacks and by the new install dir (with npm). Anyway, fixing this wasn't enough to regain the previous behavior.

To describe very quickly my app, I use nested list with custom accept, beforeDrag and dropped callbacks.

So, this is the list of my remaining problems :
* collapsing doesn't work anymore
* image of dragged item (which indicate the future drop position) doesn't always move
* (maybe it's link to the previous) it isn't possible to drop an item after the last sibling item when this item is an nonempty sublist

Well, there are probably not so much things to change in my own code. I tried to look at the diff of Readme.md but this isn't really practical and could be unreliable..

The idea is, as a simple user of this module,  it is not straightforward to have a glimpse of the big picture when 18 PR are integrated in less than 2 weeks.  All the modifications are hard to follow and is pretty difficult to see what changes and where.

Maybe, it would be a good idea to : 
* leave master clean and in sync with documentation
* make PR integration in a branch develop. like this, it would be possible and simple to explain what changed and how to update in a Changelog.md file

**Comment by h960087** [comment]
Hi,

I'm from  #351. Just saw your comment, and I found the change I missed, thanks!

I hope your problems will be easily solved too!

**Comment by JimLiu** [comment]
@jblaise  
Thanks for your advices. And I've just rollbacked to v2.1.5 ( #362  ), and will add Changelog.md to log what did it change from v2.1.5.

BTW: what do you mean about `PR`?

**Comment by jblaise** [comment]
Hum sorry use to use too much acronyms, by PR I mean Pull Request.
 
You perfectly, summarize the situation in #362 : merging too many PR make the project getting out of your control. 

maybe enforcing tests should help too. But this isn't quite easy as this module concern mostly graphical things.

**Comment by jacobscarter** [comment]
Closing this issue due to inactivity. See #815 for more information.
Please create a new issue is this still occurs with the current version of the component.
